xptifw/src/xpti_trace_framework.cpp

```diff
@@ -959,9 +959,10 @@ class Framework {
       // have 'nullptr' for both the Parent and Object only if UserData is
       // provided and the trace_point_type is function_begin/function_end.
       // This allows us to trace function calls without too much effort.
-      std::array<trace_point_type_t, 13> AllowedTypes = {
+      std::array<trace_point_type_t, 14> AllowedTypes = {
```

Comment:
also to clarify with Vasanth if there was an idea that could block me on there changes.